_topics/en/psbt.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Partially-Signed Bitcoin Transactions (PSBTs)
+
+aliases:
+  - BIP174
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Wallet Collaboration Tools
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Partially-Signed Bitcoin Transactions (PSBTs)** are a data format
+  that allows wallets and other tools to exchange information about a
+  Bitcoin transaction and the signatures necessary to complete it.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  A PSBT can be created with just information about what outputs are to
```

Comment:
This sentence isn't entirely true. PSBT shouldn't really be used with 0-input txs to be passed around for funding. We specifically avoid doing this in Core.